src/Products/Annotation/Annotator/TexStrikeoutAnnotator.cs

```diff
@@ -1,60 +1,68 @@
+            {
+                Points = new List<Point>
+                {
+                    new Point(annotationData.left, pageData.Height - annotationData.top),
+                    new Point(annotationData.left + annotationData.width, pageData.Height - annotationData.top),
+                    new Point(annotationData.left, pageData.Height - annotationData.top - annotationData.height),
+                    new Point(annotationData.left + annotationData.width, pageData.Height - annotationData.top - annotationData.height)
+                }
+            };
         }
-        
-        public override AnnotationInfo AnnotateWord()
+
+        public override AnnotationBase AnnotateWord()
         {
-            SetFixTop(true);
-            // init possible types of annotations
-            AnnotationInfo strikeoutAnnotation = InitAnnotationInfo();
+            //SetFixTop(true);
+            strikeoutAnnotation = InitAnnotationBase(strikeoutAnnotation) as StrikeoutAnnotation;
             return strikeoutAnnotation;
         }
-        
-        public override AnnotationInfo AnnotatePdf()
+
+        public override AnnotationBase AnnotatePdf()
         {
-            SetFixTop(false);
-            AnnotationInfo strikeoutAnnotation = InitAnnotationInfo();
-            strikeoutAnnotation.AnnotationPosition = new Point(annotationData.left, annotationData.top);
-            strikeoutAnnotation.PenColor = 0;
-            strikeoutAnnotation.Guid = annotationData.id.ToString();
+            //SetFixTop(false);
+            strikeoutAnnotation = InitAnnotationBase(strikeoutAnnotation) as StrikeoutAnnotation;
+            this.strikeoutAnnotation.FontColor = 0;
             return strikeoutAnnotation;
         }
-        
-        public override AnnotationInfo AnnotateCells()
+
+        public override AnnotationBase AnnotateCells()
         {
-            throw new NotSupportedException(String.Format(Message, annotationData.type));
+            throw new NotSupportedException(string.Format(Message, annotationData.type));
         }
-        
-        public override AnnotationInfo AnnotateSlides()
+
+        public override AnnotationBase AnnotateSlides()
         {
-            SetFixTop(true);
-            AnnotationInfo strikeoutAnnotation = InitAnnotationInfo();
-            strikeoutAnnotation.AnnotationPosition = new Point(annotationData.left, annotationData.top);
-            strikeoutAnnotation.PenColor = 0;
+            //SetFixTop(true);
+            strikeoutAnnotation = InitAnnotationBase(strikeoutAnnotation) as StrikeoutAnnotation;
             return strikeoutAnnotation;
         }
 
-        public override AnnotationInfo AnnotateImage()
+        public override AnnotationBase AnnotateImage()
         {
-            SetFixTop(false);
-            // init possible types of annotations
-            AnnotationInfo strikeoutAnnotation = InitAnnotationInfo();
+            //SetFixTop(false);
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Remove this commented out code.](https://app.codacy.com/gh/groupdocs-annotation/GroupDocs.Annotation-for-.NET-MVC/pullRequest?prid=5372471)